### ISSUE TITLE
can we able to take firewall backup in xml 

### ISSUE THREAD

**Issue by vTechworld** [vmware/powernsx#518]
Hi Team, 
Need to take firewall backup in XML as we do it in GUI -export configuration.  Can this be possible.

regards, 
vTechworld

**Comment by dcoghlan** [comment]
We have the ability to create the create, retrieve, and set  a DFW saved configurations, just like in the UI.

The cmdlets you are looking for are:

- Get-NsxFirewallSavedConfiguration
- New-NsxFirewallSavedConfiguration
- Remove-NsxFirewallSavedConfiguration
- Set-NsxFirewallSavedConfiguration

**Comment by vTechworld** [comment]
we can export this using export-nsxobject right

**Comment by nmbradford** [comment]
yes in theory, though, depending on your requirement, you could just go Get-NsxFirewallSavedconfiguration | set-content c:\temp\fwdump.xml
